spec/workers/webhook/destroy_worker_spec.rb

```diff
@@ -2,14 +2,13 @@
 
 RSpec.describe Webhook::DestroyWorker, type: :worker do
   let(:user) { create(:user) }
-  let(:oauth_app) { create(:application) }
-  let!(:webhook_endpoint) { create(:webhook_endpoint, user: user, oauth_application: oauth_app) }
-  let!(:other_webhook_endpoint) { create(:webhook_endpoint, oauth_application: oauth_app) }
+  let!(:webhook_endpoint) { create(:webhook_endpoint, user: user) }
+  let!(:other_webhook_endpoint) { create(:webhook_endpoint) }
   let(:worker) { subject }
 
   describe "#perform_now" do
-    it "destroys webhook by user_id and app_id" do
-      worker.perform(user.id, oauth_app.id)
+    xit "destroys webhook by user_id and app_id" do
```

Comment:
I'm disabling this spec for now as I'm planning to remove all of the webhook-related code. I feel ok about removing this since it's the only place where the worker actually seems to be called:

```
❯ rg -l Webhook::DestroyWorker
spec/workers/webhook/destroy_worker_spec.rb
```